docs/advanced-config.md

```diff
@@ -40,3 +40,13 @@ When running your application, make sure to:
 | `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE` | All versions           | `$INSTALL_DIR/integrations.json`                                               |
 
 > Some settings can be omitted on .NET (Core). For more information, see the [documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/v0.4.0-beta.1/docs/config.md#net-clr-profiler).
+
+## Splunk plugin settings
+
+Note: .NET Framework apps can read settings also from `Web.config` and `App.config`.
+
+| Environment variable                   | Description                           | Default value |
+|----------------------------------------|---------------------------------------|---------------|
+| `SPLUNK_REALM`                         | Specifies direct ingest realm.        |               |
```

Comment:
I think that we need a note about precedence - Splunk Realm and Otlp endpoint.

Reply:
Because of the wizard that generates the script for the collector, this should mention that this setting is for scenarios in that data is being sent directly to ingest.

Reply:
https://github.com/signalfx/splunk-otel-dotnet/pull/99/commits/23f860c334e4040ce9e9ec761e0b7e3026e86189